OrbitVulkanLayer/PhysicalDeviceManagerTest.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "PhysicalDeviceManager.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using ::testing::Return;
+
+namespace orbit_vulkan_layer {
+
+class MockDispatchTable {
+ public:
+  MOCK_METHOD(PFN_vkGetPhysicalDeviceProperties, GetPhysicalDeviceProperties,
+              (VkPhysicalDevice dispatchable_object));
+};
+
+TEST(PhysicalDeviceManager, ANonTrackedDeviceCanNotBeQuerried) {
```

Comment:
```suggestion
TEST(PhysicalDeviceManager, ANonTrackedDeviceCannotBeQueried) {
```

Reply:
Done, thanks.

Reply:
And also: "AnUntrackedDeviceCannotBeQueried"

Reply:
Untracked, not non-tracked

Reply:
Done, thanks!